#158429783 Create landing page with signup form

JoshuaOndieki opened the pull request:
**What does this PR do?**
Adds a landing page UI for the Ride My Way app project.

**Description of Task to be completed?**
The landing page which visitors interact with before signing in/up. The page contains a form for visitors to quickly and easily sign up to the site.

**Tasks achieved in this PR**

- [x] Landing page

- [x] Signup interface in the landing page.

**How should this be manually tested?**
Clone this repo and test locally on your computer.

**Background Context?**
The UI has no functionality.

**Resources & Libs**
[This Pivotal Tracker board](https://www.pivotaltracker.com/n/projects/2179581) was used to plan and manage the tasks.

**What are the relevant Pivotal Tracker stories?**
#158429783

**Screenshots**
![screenshot from 2018-06-19 19-23-11](https://user-images.githubusercontent.com/24773276/41610640-6e87e106-73f6-11e8-8345-2ed612a4786e.png)

JoshuaOndieki committed:
Add logo and nav to the landing page

JoshuaOndieki committed:
Add site description to the landing page

JoshuaOndieki committed:
Add signup for to the landing page

JoshuaOndieki committed:
Add background image resource

JoshuaOndieki committed:
[Feature #158429783] Add styling to the landing page

xcixor commented on UI/static/css/landingpage.css:
```diff
@@ -0,0 +1,77 @@
+    align-content: center;
+}
+
+.call-to-action{
```
Great job using classes to style elements. Makes it easy to know the group of elements that belong together

xcixor commented:
Great progress but maybe you can work on the colors a bit to enhance the look and feel..my thoughts though.

JoshuaOndieki commented:
@xcixor do you have any color in mind that I can swap?

boswellgathu commented on UI/landingpage.html:
```diff
@@ -0,0 +1,55 @@
+              <option value="passenger">Passenger</option>
+            </select>
+
+            <input type="password" placeholder="******" name="password" required>
```
please don't add the asterisk as a placeholder, the can confuse a person, just say password or any other description

boswellgathu commented on UI/static/css/main.css:
```diff
@@ -0,0 +1,71 @@
+    vertical-align: middle;
+    width: 100px;
+    display: inline-block;
+
```
Make sure to format your css properly, don't leave blank lines everywhere this makes the file look untidy and tiring to read. Also mind spacing, there should be a space between a class name and the opening curly brace.

boswellgathu reviewed (commented):
You have made good progress. I just have a few things you should look at

JoshuaOndieki committed:
Change password field placeholder text

Replace the asterisks placeholder to a more self explanatory placeholder 'passwo
rd' and 'confirm password'

JoshuaOndieki committed:
Rearrange CSS properties alphabetically

JoshuaOndieki committed:
Add consistent spacing in the css

xcixor commented:
I must admit I'm also not that good with colors but as a suggestion, think of the mood or feel your application intends to convey then visit [this site](https://color.adobe.com/explore/?filter=most-popular&time=month), type in an adjective for example 'delightful' and it gives you a list of color themes that you can use on your site

evamaina commented:
Nice one
